quality-assurance/user-acceptance-testing.md

```diff
@@ -0,0 +1,41 @@
+## What is User Acceptance Testing
+
+UAT means testing software against its business requirements i.e. does it solve the business problem as intended?
+
+## Who carries out UAT
+
+Business users are best placed to do UAT, as they are aware of the reality of their business, business processes, roles etc.  Sometimes, a tester will act in place of users.
+
+## When does UAT happen
+
+It will typically take place once a functioning system has been delivered, often after system testing.
```

Comment:
clear statement on which environment and the status of this with regards to the software versioning

Reply:
added statement about versions etc.